server/utils/nunjucksSetup.ts

```diff
@@ -22,7 +22,18 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
   app.set('view engine', 'njk')
 
   app.locals.asset_path = '/assets/'
-  app.locals.applicationName = 'Approved Premises Ui'
+
+  app.use((req, _res, next) => {
+    const service = getService(req)
+
+    if (service === 'approved-premises') {
+      app.locals.applicationName = 'Approved Premises Ui'
+    } else {
+      app.locals.applicationName = 'Temporary Accommodation Ui'
+    }
```

Comment:
We should probably call the apps just `Approved Premises` or `Temporary Accommodation`. The UI distinction is irrelevant. I've been meaning to change this for ages, was just a default name that got put in when we generated the repo from the template